dash/dash.py

```diff
@@ -1168,7 +1169,8 @@ def callback(self, output, inputs=[], state=[]):
         def wrap_func(func):
             @wraps(func)
             def add_context(*args, **kwargs):
-                output_value = func(*args, **kwargs)
+                # don't touch the comment on the next line - used by debugger
+                output_value = func(*args, **kwargs)  # %% callback invoked %%
```

Comment:
is the `%% callback invoked %%` designed for testing or there is some black magic hook underneath?